document-service/doc-api/tests/unit/models/__init__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""The Test-Suite used to ensure that the Model objects are working correctly."""
+import base64
+import uuid
+from datetime import datetime, timezone
+
+# from freezegun import freeze_time
+from sqlalchemy_continuum import versioning_manager
```

Comment:
sqlalchemy_continuum needs to be removed, if you need versioning, use the sqlalchemy package in sbc-connect-common

Reply:
Updated.